src/protocols/rdp/channels/rdpdr/rdpdr-fs-messages-file-info.c

```diff
@@ -165,9 +165,10 @@ void guac_rdpdr_fs_process_set_rename_info(guac_rdp_common_svc* svc,
             "destination_path=\"%s\"", __func__, iorequest->file_id,
             destination_path);
 
+    guac_rdp_fs* fs = device->data;
     /* If file moving to \Download folder, start stream, do not move */
     if (strncmp(destination_path, "\\Download\\", 10) == 0
-			&& !((guac_rdp_fs*) device->data)->disable_download) {
+			&& !(fs->disable_download || fs->disable_download_folder)){
```

Comment:
so the existing setting "disable_download" simply means that files are not auto downloaded to the guac server when dropped in that folder?

Reply:
```
If set to true downloads from the remote server to client (browser) will be disabled. This includes both downloads down via the hidden Guacamole menu, as well as using the special "Download" folder presented to the remote server. The default is false, which means that downloads will be allowed.

If file transfer is not enabled, this parameter is ignored.
```
https://guacamole.apache.org/doc/gug/configuring-guacamole.html